a simple unit test file is added

ibrahimbayat opened the pull request:
I added a simple unit test. It includes a function which controls the given parameters if all players are appropriate the restriction of the event. Because I am very new to the all of these unittest, api's , api's with databases, I wanted to make a simple unit test to making practise about unit testing.

KeremZaman reviewed (changes requested):
Adding a helper functon outside the test class is not a good practice. You can add this inside test class, since its name doesn't start with `test_` prefix, it won't be behaved as a test.

Also, I think class names should be according to functionality instead of the person who writes. So, I recommend changing it to `PlayerTestCase`.

For readability, please add underscore between names in function names.

In current setup, you don't test your API functions, you should write test for your API functions.

ibrahimbayat commented:
I deleted seperate test function file and added it to the test.py file. I added my test function to the inside of test class. Function names and class name are changed according to review. I couldn't achieve to write a test function for my api, but at least I wrote a simple unit test, because I am very new to all of these and I wanted to make a practice with unit testing. If you deem appropriate, I want to add my simple function to the test file.
